examples/mnist/export.py

```diff
@@ -0,0 +1,132 @@
+import sys
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import tensorflow_hub as hub
+import argparse
+""" Create a Sample TF-Hub Module using SavedModel v2.0
```

Comment:
I would rewrite this as:

"""Exporter for TF-Hub Modules in SavedModel v2.0 format.

The module has as a single signature, accepting a batch of images with shape [None, 28, 28, 1] and returning a prediction vector.

In this example, we are loading the MNIST Dataset from TFDS and training a simple digit classifier.
"""